src/ChannelAudience/Area.js

```diff
@@ -3,17 +3,19 @@
 const ChannelAudience = require('../ChannelAudience');
 
 /**
- * Audience class representing other players in the area as the sender
+ * Audience class representing other players in the same area as the sender
  */
-class Area extends ChannelAudience {
+class AreaAudience extends ChannelAudience {
```

Comment:
I tend to have a convention of class name = filename. If someone wants to namespace the class as `AreaAudience` they can do so during the `require()` e.g., `const AreaAudience  = require('./ChannelAudience/Area');`